src/app/GitUI/Editor/Diff/DiffHighlightService.cs

```diff
@@ -293,6 +293,14 @@ internal static void AddDifferenceMarkers(List<TextMarker> markers, Func<ISegmen
             markers.Add(CreateDimmedMarker(offsetRemoved + textRemoved.Length - lengthIdenticalAtEnd, lengthIdenticalAtEnd, isRemoved: true, dimBackground));
             markers.Add(CreateDimmedMarker(offsetAdded + textAdded.Length - lengthIdenticalAtEnd, lengthIdenticalAtEnd, isRemoved: false, dimBackground));
         }
+
+        return;
+
+        static ReadOnlySpan<char> LimitLength(ReadOnlySpan<char> text)
+        {
+            const int maxLength = 2000;
```

Comment:
Maybe decrease this even more?
I doubt I would scroll more than 200 characters so 500?